mappings/net/minecraft/block/BarrierBlock.mapping

```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/unmapped/C_flqucikz net/minecraft/block/BarrierBlock
+	FIELD f_rjhoxnad WATER_LOGGED Lnet/minecraft/unmapped/C_xhwijdsd;
```

Comment:
```suggestion
	FIELD f_rjhoxnad WATERLOGGED Lnet/minecraft/unmapped/C_xhwijdsd;
```
the only uses of waterlogged in the current codebase use it as one word. this also means you'll have to add it to the dictionary in `buildSrc` resources!